docs/content/introduction.md

```diff
@@ -1,31 +1,35 @@
 ---
 linkTitle: Introduction
-title: Type-safe graphql for golang
+title: Type-safe GraphQL for Go
 type: homepage
 date: 2018-03-17T13:06:47+11:00
 ---
 
 ## What is gqlgen?
 
-[gqlgen](https://github.com/99designs/gqlgen) is a golang library for building graphql servers without any fuss. gqlgen is:
+[gqlgen](https://github.com/99designs/gqlgen) is a Go library for building GraphQL servers without any fuss. gqlgen is:
 
- - Schema first: You define your API using the graphql [Schema Definition Language](http://graphql.org/learn/schema/)
- - Type safe: You should never see `map[string]interface{}` here.
- - Codegen: Let us generate the boring bits, so you can build your app quickly. 
+ - **Schema first** — Define your API using the GraphQL [Schema Definition Language](http://graphql.org/learn/schema/).
+ - **Type safe** — You should never see `map[string]interface{}` here.
+ - **Codegen** — Let us generate the boring bits, so you can build your app quickly. 
 
+[Feature Comparison]({{< ref "feature-comparison.md" >}})
 
-## Getting started
+## Getting Started
 
-First take a look at the [Getting Started]({{< ref "getting-started.md" >}}) tutorial.
+First work your way through the [Getting Started]({{< ref "getting-started.md" >}}) tutorial.
```

Comment:
**TODO** there definitely another paragraph here that mentions something about looking at Resolvers, Type Mapping etc, in roughly the order we think it's important to further learn gqlgen concepts.